pkg/trace/trace.go

```diff
@@ -198,6 +198,12 @@ func ForceTracing(ctx context.Context) context.Context {
 	return forceTracing(ctx)
 }
 
+// ForceSampleRate will force a desired sample rate for the given trace and all children
+// of said trace. The sample rate in practice will be 1/<rate>.
+func ForceSampleRate(ctx context.Context, rate uint) context.Context {
```

Comment:
This is slightly confusing, can we provide a `ExampleSampleRate`?

Reply:
How you'd invoke this in my opinion isn't confusing which is the value of `Example*` functions, but I will add more to that comment. I'll add an explicit example that maths out a sample rate